packages/tools/generator-pie-component/README.md

```diff
@@ -18,34 +18,31 @@
 
 ### Installation
 
-Install `yeoman` and the pie-component generator globally:
+Install `yeoman` globally:
 
 ```sh
-# Via NPM
-$ npm install --global yo @justeattakeaway/generator-pie-component
-
-# Using Yarn
-$ yarn add --global yo @justeattakeaway/generator-pie-component
```

Comment:
Would be good to keep the yarn global option here potentially, as I'm guessing either could be used?

Also, may be worth keeping the other docs around installation for use outside our repo, as I can see other teams wanting to spin up Lit components in the future. Could have a section for use in our monorepo and one for external use?

Reply:
the yarn option didn't work, so I'd suggest we create a small ticket to add the correct suggestions for yarn :) I don't see why using NPM at the OS/Global level would be an issue as it doesn't affect a project that uses yarn

To be clear, everything that was removed here was because it was incorrect/didn't work :)